.github/workflows/action.yml

```diff
@@ -27,7 +27,9 @@ jobs:
           check-latest: true
           cache: "pip"
       - name: Install dependencies
-        run: pip install -r requirements.txt
+        run: pip install .[dev]
+      - name: Test import
+        run: python -c "import fake_useragent; print(fake_useragent.UserAgent().random)"
```

Comment:
Why would do execute this here manually? We already run unit tests below using `tox`. That already should cover these kind of calls. right?

Reply:
The difference is that in tox, the local build is used.
That simple line ensures that the simple import and object creation works simulating what a normal user would do.
That also using the environment available in pip